init.tf

```diff
@@ -8,5 +8,9 @@ terraform {
       source  = "hashicorp/random"
       version = ">= 3.2.0"
     }
+    postgresql = {
+      source  = "cyrilgdn/postgresql"
+      version = "1.17.1"
+    }
```

Comment:
```suggestion
```

Reply:
will this work? I am not sure but applying your suggestion now..